python/py-fabio/Portfile

```diff
@@ -23,18 +23,37 @@ long_description \
 
 homepage            https://github.com/silx-kit/fabio
 
-checksums           rmd160  4006110b92008dfcf67608dbf787f577b478f85a \
-                    sha256  72ab0cb7247ec97dc552447112a54cd77b5fbe6f5ede8fc9b070e4e1c13cf01e \
-                    size    1611095
+checksums           rmd160  c75af6c617d6033783baabf5dea77b2e0c7ff89f \
+                    sha256  34f571bb8b19ee122fac7ae1d6245a92b69ae5f8f57149c7db9f6d4c307244d7 \
+                    size    726345
 
-python.versions     38 39 310
+python.versions     38 39 310 311 312
 
 if {${name} ne ${subport}} {
+    patchfiles-append \
+                    pyproject.toml.patch \
+                    requirements.txt.patch
+
+    depends_build-append \
+                    port:ninja \
+                    port:py${python.version}-cython \
+                    port:py${python.version}-wheel \
+                    port:py${python.version}-pyproject_metadata \
+                    port:py${python.version}-tomli
+
     depends_lib-append \
-                    port:py${python.version}-numpy
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-h5py \
+                    port:py${python.version}-hdf5plugin \
+                    port:py${python.version}-lxml \
+                    port:py${python.version}-Pillow
 
     test.run        yes
 
+    python.pep517   yes
```

Comment:
this is the default for Python 3.7+ so please remove

Reply:
Okay